ioctl/cmd/ws/wsprojectconfig.go

```diff
@@ -172,7 +171,9 @@ func generateProjectFile(dataSource, defaultVersion, version, vmType, codeFile,
 	verMap.Output = outputMap
 	verMaps = append(verMaps, verMap)
 
-	confMap["datasourceURI"] = dataSource
+	if dataSource != "" {
```

Comment:
seems like unnecessary changes

Reply:
if `dataSource` is nil, the value of key `datasourceURI` in the json is `""`, and it is not expected.
if `dataSource` is nil, the key `datasourceURI` should not exist in the json.